portfolio/src/main/webapp/script.js

```diff
@@ -13,20 +13,6 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random fact about Andrea to the page.
```

Comment:
Why is this block being deleted? Is script.js being branched somehow? It's been awhile since I used GitHub.